app/serializers/spree/api/v2/display_serializer.rb

```diff
@@ -2,7 +2,7 @@ module Spree
   module Api
     module V2
       class DisplaySerializer < BaseSerializer
-        attributes :id, :name, :screen_size, :orientation, :vendor_id, :display_type, :products_array, :active
+        attributes :id, :name, :screen_size, :orientation, :vendor_id, :display_type, :products_array, :active, :qr_code_url
```

Comment:
why do we have qr code url?

Reply:
this is also something generatable like we did on product right?

Reply:
I am following the similar pattern for attribute used in the display product model and the qr is also generatable as in product.

Reply:
maile product ma pani url hatauna ra dynamic ganerate garna bhaneko thyea..why do we have url in db?